cli/render-dedupe-check-issues/package.json

```diff
@@ -0,0 +1,44 @@
+{
+  "name": "@pnpm/cli.render-dedupe-check-issues",
```

Comment:
put this package to a new directory called `dedupe` and rename to `@pnpm/dedupe.issues-renderer`

Reply:
Done in https://github.com/gluxon/pnpm/commit/c95ed844d17fbf4fb4521a38e2cbf8f51538aa75.